src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -91,6 +93,7 @@ class NotebookLauncherContent extends Component {
         return cluster
       } else if (status === 'Stopped') {
         await Jupyter.cluster(googleProject, clusterName).start()
+        refreshClusters()
```

Comment:
TOL: It's worth noting that there will be two independent 'processes' polling simultaneously, which may result in some extra overlapping requests. Not a problem, but at some point it would be interesting to see if we could create a unified approach, since I doubt this is the last time we'll do something like this.

Reply:
Indeed. I'd thought about maybe memoizing the response with like a 500ms timeout, but that seemed to create more problems than it solved.